projects/storefrontlib/shared/components/table/table.component.html

```diff
@@ -19,7 +19,7 @@
             [cxOutlet]="getDataOutletRef(cell)"
             [cxOutletContext]="getDataOutletContext(cell, item)"
           >
-            {{ item[cell] }}
+            {{ $any(item)[cell] }}
```

Comment:
in this case, I believe we don't need a user defined guard since the data is generic